installer.tpl.sh

```diff
@@ -217,6 +217,13 @@ install_apt ()
   curl_check
   gpg_check
 
+  # For 3 series we use static builds and we can use one repository
+  # for all deb-based and rpm-based systems
+  if [ "${ver}" = 3 ]; then
+    os="linux-deb"
```

Comment:
This will set such `os` and `dist` for modules as well, but it shouldn't. We have static build packages only for tarantool.

Reply:
I fixed it in another way.